File: cpp/03.cpp
#include <iostream>
#include <cmath>
#include <chrono>

int main() {
    // Start the timer
    auto start = std::chrono::high_resolution_clock::now();

    // Perform scientific computations and mathematical calculations here
    double result = 0;
    for (int i = 0; i < 1000000; ++i) {
        result += std::sin(i) * std::cos(i);
    }

    // Stop the timer
    auto end = std::chrono::high_resolution_clock::now();
    std::chrono::duration<double> duration = end - start;

    // Display the result and the time taken
    std::cout << "Result: " << result << std::endl;
    std::cout << "Time taken: " << duration.count() << " seconds" << std::endl;

    return 0;
}

File: python/03.py
import time
import math

# Start the timer
start = time.time()

# Perform scientific computations and mathematical calculations here
result = 0
for i in range(1000000):
    result += math.sin(i) * math.cos(i)

# Stop the timer
end = time.time()
duration = end - start

# Display the result and the time taken
print("Result:", result)
print("Time taken:", duration, "seconds")
